valhalla/midgard/tiles.h

```diff
@@ -147,7 +147,10 @@ template <class coord_t> class Tiles {
     if (x == tilebounds_.maxx()) {
       return ncolumns_ - 1;
     } else {
-      float col = (x - tilebounds_.minx()) / tilesize_;
```

Comment:
this was the actual bug. it assigned a column of 1440 for a node, which is 1 past the tilesystem's max of 1399.

Reply:
wow years on we are still finding places where we have `float`s floating around..

Reply:
Btw. is there any point in having `Tiles` as a template class? I can find only one place where it is used not with `PointLL` and it is tests...